contracts/src/Teleporter/ReceiptQueue.sol

```diff
@@ -49,32 +49,34 @@ library ReceiptQueue {
      */
     function dequeue(TeleporterMessageReceiptQueue storage queue)
         internal
-        returns (TeleporterMessageReceipt memory result)
+        returns (TeleporterMessageReceipt memory)
     {
         uint256 first_ = queue.first;
         require(queue.last != first_, "ReceiptQueue: empty queue");
-        result = queue.data[first_];
+        TeleporterMessageReceipt memory receipt = queue.data[first_];
         delete queue.data[first_];
         queue.first = first_ + 1;
+        return receipt;
     }
 
     /**
      * @dev Returns the outstanding receipts for the given chain ID that should be included in the next message sent.
      */
     function getOutstandingReceiptsToSend(TeleporterMessageReceiptQueue storage queue)
         internal
-        returns (TeleporterMessageReceipt[] memory result)
+        returns (TeleporterMessageReceipt[] memory)
```

Comment:
I agree that we should remove named return parameters for the sake of readability, but I'm curious if there are any gas benefits one way or the other? The OZ report does not specify.

Reply:
from the gas reports didn't notice gas changes, think that makes sense since under the hood the named return parameters are likely the same as declaring a new local variable.